<launch>

  <!-- Start openpose_ros -->
  <node pkg="openpose_ros" type="openpose_ros_node" name="openpose_ros_node" output="screen">
    <rosparam command="load" file="$(find openpose_ros)/config/config.yaml"/>
  </node>

</launch>